layouts/adminLayout.tsx

```diff
@@ -15,13 +17,41 @@ const AdminLayout: NextPage<InferProps<typeof AdminLayoutProps>> = ({
   children,
 }) => {
   const router = useRouter();
+  const [navSwitch, setNavSwitch] = useState(false);
+  const [focused, setFocused] = useState<string | null>(null);
+  const [isOpen, setIsOpen] = useState(false);
+
+  useEffect(() => {
+    if (window.innerWidth > 640 && window.innerWidth < 840) {
+      setNavSwitch(true);
+    }
+  }, [navSwitch, setNavSwitch]);
+
+  const handleNavSwitch: () => void = (): void => {
```

Comment:
why returning a function

Reply:
```suggestion
  const handleNavSwitch = (): void => {
  }
```

Reply:
please fix this accordingly

Reply:
there is no point for the function to return another function

Reply:
Fixed it